frontend/scripts/componentSchemas/fileUtils.ts

```diff
@@ -0,0 +1,17 @@
+import type { AppFrontendVersion } from './version';
+import { versionSettings } from './version';
+import path from 'path';
+import fs from 'fs';
+
+export const writeToFile = (name: string, data: any, version: AppFrontendVersion) => {
```

Comment:
Hmm, I think the types would have to be something general like `Record<String, any>`, since we are dealing with JSON schemas or parts of JSON schemas. Would that add any value?

Reply:
I have now replaced all instances of `any`.
It took a while, but I learned some things as well. Thanks for the challenge 😄 
I even discovered an error in the script - see its own comment.

Reply:
That's great @ErlingHauan, well done! 👏 
An important thing with using types is that we can spot these errors easier, great job!